### PR TITLE
Add the first version of the timeline section

### PR DESCRIPTION
**Pull request by mwydmuch**
Changes:
- Added a timeline section that can accept arguments for better modularity.
- Small style changes here and there.
- For now, replaced all sponsors with ML in PL's logo / URL.

### CODE REVIEW

File: _data/mlss-timeline.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+- date: 3 March
+  description: Registration open
+
+- date: 8 April
+  description: Registation deadline
```
Comment (Podkamienna) [inline _data/mlss-timeline.yml:5]: Typo: Registation -> Registration

File: _data/mlss-timeline.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- date: 28 April
+  description: Acceptence notifications
```
Comment (Podkamienna) [inline _data/mlss-timeline.yml:8]: Typo: Acceptence -> Acceptance

File: _includes/css/mlinpl.css [hunk 3]
```diff
@@ -199,6 +203,10 @@ a:focus {
     }
 }
 
+.navbar-nav, .navbar-brand{
```
Comment (Podkamienna) [inline _includes/css/mlinpl.css:206]: Would it be possible, to make this view of the navbar look better? 
image

File: _includes/mlss/about.html [hunk 4]
```diff
@@ -9,7 +9,8 @@ <h2>/ About</h2>
                 research-oriented graduate students. The school features a line-up of internationally recognised
                 researchers who will talk with enthusiasm about their subjects. Our goal is to provide a unique
                 opportunity to learn from and connect with the leading experts in the scenic setting of the historical
-                city of Cracow, Poland.
+                city of Cracow, Poland. 
+                This school is next edtion of <a href="https://mlss.mlinpl.org" style="color: #000;"><span class="logo"><span class="main-name">MLSS<span class="superscript" style="color: #a5619b !important">N</span></span> <span class="sub-name">2022</span></span></a>.
```
Comment (Podkamienna) [inline _includes/mlss/about.html:13]: Would ''the next edition' sound better?

Comment (mwydmuch) [reply _includes/mlss/about.html]: Yes, changed!

File: _data/mlss-timeline.yml (hunk 2 above)
Comment (mwydmuch) [reply _data/mlss-timeline.yml]: Thank you, fixed

File: _data/mlss-timeline.yml (hunk 1 above)
Comment (mwydmuch) [reply _data/mlss-timeline.yml]: Thank you, fixed

File: _includes/css/mlinpl.css (hunk 3 above)
Comment (mwydmuch) [reply _includes/css/mlinpl.css:206]: Yes, we need to improve it, but this is not the aim of this PR :)

**Review by Podkamienna — APPROVED** [review]

**Pull request merged**